deploy/plays/tasks/dockerbuild.yml

```diff
@@ -0,0 +1,18 @@
+---
+  - name: Build main server docker image
+    become: yes
+    docker_image:
+      name: "main_server"
+      path: "../../main_server"
```

Comment:
Docker build happens before copy.sh in our setup. This is needed keeping in mind that docker image load may possibly require more than one try. Install directory does not exist before copy, which is the part of playbook.yml. And hence, the dockerfile from the local directories are used instead of the install directory.